lib/abbreviato/abbreviato.rb

```diff
@@ -0,0 +1,22 @@
+module Abbreviato
+  DEFAULT_OPTIONS = {
+      max_length: 30,
+      tail: "..."
+  }.freeze
+
+  # Truncates the source XML string and returns the truncated XML. It will keep a valid XML structure
```

Comment:
should this be modified to include info about the return of the `.truncated` field?